### PR TITLE
Pydantic 1.0 Changes

### PR DESCRIPTION
**Pull request by dgasmith**
## Description
Updates and fixes Pydantic 1.0 changes and deprecation warnings. This requires a new QCElemental release before merging.

## Status
- [ ] Changelog updated
- [ ] Ready to go

### CODE REVIEW

**Comment by mattwelborn** [comment]
Are there any blockers on this now that qcel 0.11 is released?

**Comment by dgasmith** [comment]
Should be good to go, restarting tests now.

**Comment by dgasmith** [comment]
@mattwelborn Please review and merge.

File: qcfractal/procedures/procedures.py [hunk 1]
```diff
@@ -126,6 +126,8 @@ def parse_input(self, data):
             inp = record.build_schema_input(mol, keywords)
             inp.extras["_qcfractal_tags"] = {"program": record.program, "keywords": record.keywords}
```
Comment (mattwelborn) [inline qcfractal/procedures/procedures.py]: leftover debug prints?

File: qcfractal/interface/collections/dataset_view.py [hunk 2]
```diff
@@ -418,7 +418,7 @@ def _deserialize_field(field: str) -> Any:
     def _serialize_data(data: Any) -> np.ndarray:
         # h5py v3 will support bytes,
         # but for now the workaround is variable-length np unit8
-        return np.fromstring(serialize(data, 'msgpack-ext'), dtype='uint8')
+        return np.frombuffer(serialize(data, 'msgpack-ext'), dtype='uint8')
```
Comment (mattwelborn) [inline qcfractal/interface/collections/dataset_view.py]: Good catch!

**Comment by mattwelborn** [comment]
LGTM, I think the pin in setup.py needs to be updated.

**Pull request merged**